integration-cli/docker_cli_run_test.go

```diff
@@ -101,12 +101,9 @@ func (s *DockerSuite) TestRunEchoStdoutWitCPUQuota(c *check.C) {
 		c.Errorf("container should've printed 'test'")
 	}
 
-	cmd := exec.Command(dockerBinary, "inspect", "-f", "{{.HostConfig.CpuQuota}}", "test")
-	out, _, err = runCommandWithOutput(cmd)
-	if err != nil {
-		c.Fatalf("failed to inspect container: %s, %v", out, err)
+	if out, err = inspectField("test", "HostConfig.CpuQuota"); err != nil {
+		c.Errorf("%v", err)
```

Comment:
My mistake, but seem there are lots of `c.Errorf` in that file, maybe see it in another PR.